Minor changes

MeuhV1 opened the pull request:
Updated strategy texts
Fixed python exception in WB stability maps editor
Very minor code refactoring
Incorporated GP naming in WB unit placing

dguenms commented on Assets/Python/GreatPeople.py:
```diff
@@ -65,7 +65,7 @@ def getName(unit):
 	return utils.getRandomEntry(lAvailableNames)
 	
-def onGreatPersonBorn(unit, iPlayer, city):
+def onGreatPersonBorn(unit, iPlayer, city, bWB = False):
```
Can you rename the new argument to something that is meaningful to the method itself? I would prefer something like `bAnnounceBirth = True`.

Commit:
Updated strategy texts
Fixed python exception in WB stability maps editor
Very minor code refactoring
Incorporated GP naming in WB unit placing

MeuhV1 commented:
Updated PR with your comment.
